### ISSUE TITLE
Can there be a feature to preview the captured image first, and maybe then run the onFile method ?

### ISSUE THREAD

**Issue by Chiragjot** [Gabul-DEV/camera_camera#93]

**Comment by gabuldev** [comment]
Yes! I Will create a new method to retorn a Stream<Image>. I think this solves the problem. What do you think?

**Comment by Chiragjot** [comment]
yes could do that, what I did when I used the plugin was push a new screen inside the onfile method and previewed the image on that screen and also provide the cancel and done buttons on that screen itself, but yea this solution was kind of particular to my usage. You can add something similar in a different method, would be a great feature to have IMO.

**Comment by gabuldev** [comment]
Can you test this branch? feature/listen-image
